ManageWikiSettings.php

```diff
@@ -363,6 +362,51 @@
 		'section' => 'edit',
 		'help' => 'Actions that can be restricted.',
 	],
+	'wgCommentStreamsEnableTalk' => [
+		'name' => 'Enable in talk namespaces',
+		'from' => 'commentstreams',
+		'restricted' => false,
+		'type' => 'check',
+		'overridedefault' => false,
+		'section' => 'edit',
+		'help' => 'Enable in talk namespaces',
```

Comment:
I need suggestions on the text

Reply:
This is fine as it's a description, the name of this should indicate this is for comment streams.